openlibrary/templates/type/edition/view.html

```diff
@@ -478,7 +478,7 @@ <h3 class="list-header collapse">$_("ID Numbers")</h3>
     $# pages like /books/ia:foo00bar are fake records created from metadata API.
     $# Adding them to lists doesn't work. Disabling it to avoid any issues.
     $if "lists" in ctx.features and not page.is_fake_record():
-      <h2 class="home-h2"><a href="$page.url()/lists">$_('Lists containing this Book')</a></h2>
+      <h2 class="home-h2" id="lists"><a href="$page.url()/lists">$_('Lists containing this Book')</a></h2>
```

Comment:
I wonder if we want to use `name` here instead of `id` to avoid any possible styling collisions. @jimchamp ?

Otherwise, changes LGTM!

Reply:
I think that using `id` is fine.  I didn't see any CSS rules for `#lists` after doing a quick search, but it may be better to be more specific with the ID.  Something like `lists-section` or `lists-details` would be great.  I'll let you decide which, @homewardgamer.